recipes/agent_windows.rb

```diff
@@ -7,4 +7,33 @@
 #
 # Recipe to install and configure the New Relic Infrastructure agent on Windows
 #
-raise 'Chef support for the Windows New Relic Infrastructure agent not yet available!'
+# Chef uses sha256 checksums, to generate this, I downloaded the file
+# and on OS X did 'shasum -a 256 newrelic-infra.msi' YMMV on other
+# platforms. It'd be *really* cool if NewRelic could provide an API
+# that you could ask for checksum values, or have checksums published
+# and available in some other way.
+#
+# TODO: make install location configurable
+
+windows_package 'newrelic-infra' do
+  action :install
+  source node['newrelic-infra']['windows_source'] if node['newrelic-infra']['windows_source']    
```

Comment:
These should just be `nil` if these attributes are not defined which, would just be the same as not including them at all. Is there a reason those are here or can they be removed?